mobile/src/main/java/ch/epfl/sdp/mobile/ui/tournaments/Contest.kt

```diff
@@ -0,0 +1,18 @@
+package ch.epfl.sdp.mobile.ui.tournaments
+
+import kotlin.time.Duration
+
+/** The button to create a new tournament. */
+interface Contest {
```

Comment:
I would name this `ContestInfo`, to better distinguish from composables which might be named `Contest`.